app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -344,8 +344,24 @@ export default class SampleViewV2 extends React.Component {
 
   getTaxonMetricValue = (row, metric) => {
     let parsedMetric = metric.split(":");
+    let metricDecimalPlaces = this.getDecimalPlacesFromMetric(parsedMetric[1]);
```

Comment:
I wonder if we should just do this on `fetchSampleReportData` just after we loaded the raw data. That way we would do it just once. wdyt?

Reply:
Yeah definitely I think it's a great idea!